integration-tests/api/__tests__/admin/product.js

```diff
@@ -909,6 +909,137 @@ describe("/admin/products", () => {
     })
   })
 
+  describe("GET /admin/products/:id", () => {
+    const productId = "testing-get-product"
+
+    beforeEach(async () => {
+      await simpleProductFactory(dbConnection, {
+        id: productId,
+        variants: [
+          {
+            title: "Test variant",
+            prices: [
+              {
+                currency: "usd",
+                amount: 100,
+              },
+            ],
+          },
+        ],
+      })
+
+      await adminSeeder(dbConnection)
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      await db.teardown()
+    })
+
+    it("should get a product with default relations", async () => {
+      const api = useApi()
+
+      const res = await api
+        .get(`/admin/products/${productId}`, adminHeaders)
+        .catch((err) => {
+          console.log(err)
+        })
+
+      const keysInResponse = Object.keys(res.data.product)
+
+      expect(res.status).toEqual(200)
+      expect(res.data.product.id).toEqual(productId)
+      expect(keysInResponse).toEqual(
+        expect.arrayContaining([
+          // fields
+          "id",
+          "created_at",
+          "updated_at",
+          "deleted_at",
+          "title",
+          "subtitle",
+          "description",
+          "handle",
+          "is_giftcard",
+          "status",
+          "thumbnail",
+          "weight",
+          "length",
+          "height",
+          "width",
+          "hs_code",
+          "origin_country",
+          "mid_code",
+          "material",
+          "collection_id",
+          "type_id",
+          "discountable",
+          "external_id",
+          "metadata",
+
+          // relations
+          "categories",
+          "collection",
+          "images",
+          "options",
+          "profiles",
+          "profile",
+          "profile_id",
+          "sales_channels",
+          "tags",
+          "type",
+          "variants",
```

Comment:
Suggestion : by default here variants.prices should also be present. Should we add that?

Reply:
Prices are nested on variants, so they are not present when using `Object.keys(product)`

Reply:
What i ve done in the other tests is that the nested known props i ve pushed them in the expected variable. If one of them are not present anymore i think it is better that the tests fails so that we prevent unwanted relation removal. Wdyt?

Reply:
Can I get you to push these changes?